src/show.jl

```diff
@@ -11,6 +11,15 @@ function print_graded_results(io::IO, ts::GradedTestSet, depth_pad=0)
     return print_graded_results(io, result(ts), depth_pad)
 end
 
+# Convert number to integer if reasonable to avoid floating point representation
+function display_number(n::Number)
+    if n == round(n)
+        return Int(n)
+    else
+        return n
+    end
+end
+
```

Comment:
Is this really needed?

Reply:
It is not needed and purely a cosmetic feature. I thought it'd look nicer if it the output would print `3` instead of `3.0` as described in the screenshots above. Functionally, however, it's not needed

Reply:
If everything is uniformly `Float64`, it might not be needed anymore!